components/support/rand_rccrypto/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+pub use rand_core;
+use rand_core::{impls, CryptoRng, Error, RngCore};
+
+pub struct RcCryptoRng;
+
+impl RngCore for RcCryptoRng {
+    fn next_u32(&mut self) -> u32 {
+        impls::next_u32_via_fill(self)
+    }
+
+    fn next_u64(&mut self) -> u64 {
+        impls::next_u64_via_fill(self)
+    }
+
+    fn fill_bytes(&mut self, dest: &mut [u8]) {
+        self.try_fill_bytes(dest).unwrap()
```

Comment:
nit: we should at the very least panic with a descriptive message here. The `OsRng` impl is an example of more elaborate error handling, which includes a retry loop:

  * https://docs.rs/rand/0.5.0/src/rand/rngs/os.rs.html#120

But that might be overkill for our purposes.